packages/db-ui-elements-stencil/src/components/db-breadcrumb/db-breadcrumb.tsx

```diff
@@ -24,7 +24,7 @@ export class DbBreadcrumb {
   /**
    * an optional [`aria-label`-attribute](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label)
    */
-  @Prop({ reflect: true, attribute: 'aria-label' }) ariaLabel: string;
+  @Prop({ reflect: true }) aria: string;
```

Comment:
I could think about renaming that property to `label` instead of `aria`, because if we need to remove one of its previous names parts, label would be still more accurate, but instead [`aria` is the "namespace" for a whole bunch of further attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes).

Reply:
renamed it to alabel

Reply:
@nmerget ginge auch arialabel anstelle dessen (also wie das Standard-Attribut, aber eben ohne Bindestrich)?

Reply:
I think https://www.youtube.com/watch?v=SNdcFPjGsm8 is one of the craziest songs ever and I couldn't understand it until now. Doing the same thing over an over haha

fixed